tools/autograph-monitor/monitor.go

```diff
@@ -190,32 +182,9 @@ func makeAuthHeader(req *http.Request, user, token string) string {
 	return auth.RequestHeader()
 }
 
-// send a message to a predefined sns topic
-func sendSoftNotification(id string, format string, a ...interface{}) error {
-	if ts, ok := softNotifCache[id]; ok {
-		// don't send dup notifications for 24 hours
-		if ts.Add(24 * time.Hour).After(time.Now()) {
-			log.Printf("silencing soft notification ID %s", id)
-			return nil
-		}
-	}
-	if os.Getenv("LAMBDA_TASK_ROOT") == "" || os.Getenv("AUTOGRAPH_SOFT_NOTIFICATION_SNS") == "" {
-		// We're not running in lambda or the conf isnt ready so don't try to publish to SQS
-		log.Printf("soft notification ID %s: %s", id, fmt.Sprintf(format, a...))
-		return nil
-	}
-
-	svc := sns.New(session.New())
-	params := &sns.PublishInput{
-		Message:  aws.String(fmt.Sprintf(format, a...)),
-		TopicArn: aws.String(os.Getenv("AUTOGRAPH_SOFT_NOTIFICATION_SNS")),
-	}
-	_, err := svc.Publish(params)
-	if err != nil {
-		return err
-	}
-	log.Printf("Soft notification send to %q with body: %s", os.Getenv("AUTOGRAPH_SOFT_NOTIFICATION_SNS"), params.Message)
-	// add the notification to the cache
-	softNotifCache[id] = time.Now()
-	return nil
+// https://docs.datadoghq.com/integrations/amazon_lambda/
+func sendDatadogMetric(value int, mType, mName string, tags []string) {
+	mName = strings.Replace(mName, " ", "_", -1)
```

Comment:
According to [this](https://help.datadoghq.com/hc/en-us/articles/203764705-What-are-valid-metric-names-) metric names shouldn't have '-' either, but it will be autoconverted away for us.